pkg/ebpf/bpf_client.go

```diff
@@ -291,6 +293,8 @@ type bpfClient struct {
 	IngressProgToPodsMap *sync.Map
 	// Stores the Egress eBPF Prog FD to pods mapping
 	EgressProgToPodsMap *sync.Map
+	// Stores pod to attachprobes lock mapping
+	PodToAttachProbesLock *sync.Map
```

Comment:
I think, a more intuitive name for this variable can be `AttachProbesToPodLock` 

I was getting confused with `PodTo` as start of the variable name.

Reply:
sure. Name wise I was going for mapping pod -> lock but this "AttachProbesToPodLock" might be more readable

Reply:
updated